CONTRIBUTING.md

```diff
@@ -26,7 +26,10 @@ We're always looking for more opinions on discussions in the issue tracker. It's
 - For ambitious tasks, you should try to get your work in front of the community for feedback as soon as possible. Open a pull request as soon as you have done the minimum needed to demonstrate your idea. At this early stage, don't worry about making things perfect, or 100% complete. Add a `[WIP]` prefix to the title, and describe what you still need to do. This lets reviewers know not to nit-pick small details or point out improvements you already know you need to make.
 - Don't include unrelated changes
 - New features should be accompanied with tests and documentation
-- Use a clear and descriptive title for the pull request and commits
+- Use a clear and descriptive title for the pull request and commits, following
+  [Angular's git commit guidelines](https://github.com/angular/angular.js/blob/master/DEVELOPERS.md#-git-commit-guidelines).
+  You can run `npx sgc` instead of `git commit` to commit and you will be asked questions to help format your commit
+  message properly.
```

Comment:
So do you _have_ to run `npx sgc` to commit? What if you just know the format and want to do it yourself? Is there anything that will catch a bad format before you push? I think we need to be clearer about this kind of thing if possible.

Reply:
You can just know the format. The tool just makes it easier. I'll add some more text to make this more clear.